src/H5FDtest.c

```diff
@@ -89,7 +89,10 @@ H5FD__supports_swmr_test(const char *vfd_name)
 
     FUNC_ENTER_NOAPI_NOINIT_NOERR
 
-    if (!vfd_name || !strcmp(vfd_name, "") || !strcmp(vfd_name, "nomatch"))
```

Comment:
No need for the "nomatch" case. It was only checked in a few places where we can just as easily return "sec2" for the same behavior.